grub.default

```diff
@@ -3,31 +3,41 @@ GRUB_TIMEOUT=5
 # Device specific configs
 DEVICENAME=$(cat /sys/devices/virtual/dmi/id/product_name)
 # OXP Devices
-OXP_LIST="ONE XPLAYER:ONEXPLAYER 1 T08:ONEXPLAYER 1S A08:ONEXPLAYER 1S T08:ONEXPLAYER mini A07:ONEXPLAYER mini GA72:ONEXPLAYER mini GT72:ONEXPLAYER GUNDAM GA72:ONEXPLAYER 2 ARP23"
+OXP_LIST="ONE XPLAYER:ONEXPLAYER 1 T08:ONEXPLAYER 1S A08:ONEXPLAYER 1S T08:ONEXPLAYER mini A07:ONEXPLAYER mini GA72:ONEXPLAYER mini GT72:ONEXPLAYER GUNDAM GA72:ONEXPLAYER 2 ARP23:ONEXPLAYER Mini Pro"
 # AYANEO AIR Devices
 AIR_LIST="AIR:AIR Pro"
 # AYANEO Gen 1 Devices
-AYA_LIST="Next:Some fucking thing i don't own right now"
+AYA_LIST="NEXT:NEXT Pro:NEXT Advance:AYANEO NEXT:AYANEO NEXT Pro:AYANEO NEXT Advance:AYA NEO Fouder:AYA NEO 2021:AYANEO 2021:AYANEO 2021 Pro:AYANEO 2021 Pro:AYANEO 2021 Pro Retro Power"
```

Comment:
"AYA NEO Fouder" needs to be "AYANEO Founder" maybe?

Reply:
![WK2{}P 7U4PX`$V0QSRFGHX](https://user-images.githubusercontent.com/1498746/201824810-a16bb69a-e5ae-48c9-9566-f30582637f5a.jpg)

Reply:
I don't think I'm mistaken

Or we could add both